web-client/src/presenter/actions/TrialSessionWorkingCopy/getFormattedTrialSessionDetailsAction.js

```diff
@@ -0,0 +1,13 @@
+import { state } from 'cerebral';
+
+/**
+ * getFormattedTrialSessionDetailsAction
+ *
+ * @param {object} providers.get the cerebral get function
+ * @returns {Object} formattedTrialSessionDetails
+ */
+
+export const getFormattedTrialSessionDetailsAction = ({ get }) => {
```

Comment:
🔧 Same thing here, can the action that uses `formattedTrialSessionDetails` just pull that value off of state?